mono/utils/mono-error-internals.h

```diff
@@ -113,6 +113,12 @@ mono_error_set_invalid_operation (MonoError *error, const char *msg_format, ...)
 void
 mono_error_set_exception_instance (MonoError *error, MonoException *exc);
 
+void
+mono_error_set_invalid_program (MonoError *oerror);
```

Comment:
I'll add the message argument, but in the single place where we set this exception, the message is NULL. Should we change this?

https://github.com/mono/mono/blob/master/mono/metadata/class.c#L6690